code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/internal/rancher/RancherIdentitySearchProvider.java

```diff
@@ -94,7 +93,7 @@ public Identity getIdentity(String id, String scope) {
 
     @Override
     public boolean isConfigured() {
-        return !SecurityConstants.SECURITY.get();
+        return true;
```

Comment:
I don't understand your explanation.  I think you're missing a lot of punctuation in your sentence.